bash/convert_dtm.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# was run using db-knownprojects-data dev container
+# https://manpages.debian.org/experimental/postgis/shp2pgsql.1.en.html
+# http://www.bostongis.com/pgsql2shp_shp2pgsql_quickguide_20.bqg
+set -e
+echo "Hi! Convert DTM shapefile to PostgreSQL..."
+
+# ogr2ogr dof_dtm_2/dof_dtm_2.sql dof_dtm_1/dof_dtm_1.shp
+shp2pgsql -d -D -I -g wkb_geometry -s 4326 dof_dtm_1/dof_dtm_1.shp public.dof_dtm >dof_dtm_2/dof_dtm_2.sql
```

Comment:
No problem with this, just curious - rationale for this being its own file rather than say a function in config.sh?

Reply:
ah, because these aren't intended to be run by other scripts during this build. and they're in 2 separate scripts rather than 1 because it was important to inspect the intermediate file, they take so long to run, and I didn't setup a single environment which could run both (hence the dev container notes at the tops of the files)